src/app/pages/psbt-request/components/psbt-decoded-request/psbt-decoded-request.tsx

```diff
@@ -0,0 +1,37 @@
+import { useState } from 'react';
+
+import { Box, Stack, color } from '@stacks/ui';
+
+import { PsbtDecodedRequestAdvanced } from './psbt-decoded-request-views/psbt-decoded-request-advanced';
+import { PsbtDecodedRequestSimple } from './psbt-decoded-request-views/psbt-decoded-request-simple';
+import { PsbtDecodedRequestViewToggle } from './psbt-decoded-request-views/psbt-decoded-request-view-toggle';
+
+interface PsbtDecodedRequestProps {
+  psbt: any;
+}
+export function PsbtDecodedRequest({ psbt }: PsbtDecodedRequestProps) {
+  const [showAdvancedView, setShowAdvancedView] = useState(false);
+
+  return (
+    <Box minHeight="260px">
```

Comment:
min height on stack?

Reply:
Same here, think others set this so let me check if needed.